src/adr/articles/0012-aws-xray.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: layouts/adr.njk
+title: "AWS X-Ray"
+weight: 1
+date: 2022-03-21
+review_in: 12 months
+tags:  
+    - adr
+    - observability
+    - tools
+    - language
+areas_of_coverage: ["Digital Service"]
+status: "accepted"
+contributors: ["John Nolan"]
+adr_number: "0012"
+---
+
+## Context
+
+We want a way to trace, analyse and debug our distributed systems across our cloud architecture.
+
+This will give us the ability to
+
+* Find bottle necks in our services
+* Discover hidden errors within our stack
+* Identify areas of improvement
+* Justify continuous improvement work
+
+## Technical
+
+### Developer Knowledge - How well known is this in our current skill sets
+
+**Overall**: 5/10
+Our teams are starting to use this functionality in their services.
```

Comment:
Maybe add something here on what we need to do to boost this level of knowledge. We've talked about an observability COP before.